### PR TITLE
KIPSU-25907 - Add README

### PR DESCRIPTION
**Pull request by Dsorthep**
This is a PR to add a README to this project. This README has a description of the repo along with a link to the Services doc, setup instructions with a link to download IntelliJ IDEA, and commands to start and test the application.

### CODE REVIEW

**Comment by mlafonta** [comment]
If you name your PR starting with "KIPSU-25907 - {name}" it will automatically link to Jira the same as bitbucket

**Review by mlafonta — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
-## Micronaut 3.7.4 Documentation
+## telegraph (the Message Provider Platform)
+**telegraph** is an integration platform for messaging providers. This monorepo is written in Kotlin with a Micronaut framework and Apache Camel to handle queueing.
```
Comment (kronning6) [inline README.md:2]: Lets scrap `integration` from this description and change the last sentence to "This monorepo is written in Kotlin with the Micronaut framework and Apache Camel."

File: README.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 
-- [User Guide](https://docs.micronaut.io/3.7.4/guide/index.html)
-- [API Reference](https://docs.micronaut.io/3.7.4/api/index.html)
-- [Configuration Reference](https://docs.micronaut.io/3.7.4/guide/configurationreference.html)
-- [Micronaut Guides](https://guides.micronaut.io/index.html)
----
+For more information on **telegraph**: [Telegraph (MPP)](https://kipsudev.atlassian.net/wiki/spaces/EN/pages/2969698317/Telegraph+MPP)
+### Setup
+To get started clone this repo. Downloading Intellij IDEA (IDE) is recommended for this Kotlin based application.
```
[the remark this answers is not in the document]
Comment (kronning6) [reply README.md:6]: Lets capitalize the J in `IntelliJ`

File: README.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
-## Micronaut 3.7.4 Documentation
+## telegraph (the Message Provider Platform)
+**telegraph** is an integration platform for messaging providers. This monorepo is written in Kotlin with a Micronaut framework and Apache Camel to handle queueing.
 
-- [User Guide](https://docs.micronaut.io/3.7.4/guide/index.html)
-- [API Reference](https://docs.micronaut.io/3.7.4/api/index.html)
-- [Configuration Reference](https://docs.micronaut.io/3.7.4/guide/configurationreference.html)
-- [Micronaut Guides](https://guides.micronaut.io/index.html)
----
+For more information on **telegraph**: [Telegraph (MPP)](https://kipsudev.atlassian.net/wiki/spaces/EN/pages/2969698317/Telegraph+MPP)
```
Comment (kronning6) [inline README.md:4]: Lets spell out Message Provider Platform

File: README.md [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
+### Setup
+To get started clone this repo. Downloading Intellij IDEA (IDE) is recommended for this Kotlin based application. 
+### Application
+`./gradlew run` - Starts the application on port 8080.
```
Comment (kronning6) [inline README.md:8]: This will change with Matt's Docker PR but lets change this to `./gradlew run --continuous`

**Review by kronning6 — CHANGES_REQUESTED** [review]
Please update the title to have the ticket number

**Comment by Dsorthep** [comment]
This is a PR to add a README to this project. This README has a description of the repo along with a link to the Services doc, setup instructions with a link to download IntelliJ IDEA, and commands to start and test the application.

**Review by mlafonta — APPROVED** [review]
lgtm

**Review by kronning6 — APPROVED** [review]

**Review by matthewbouc — APPROVED** [review]

**Pull request merged**